app/controllers/course/forum/posts_controller.rb

```diff
@@ -28,6 +29,7 @@ def update
 
   def vote
     @post.cast_vote!(current_user, post_vote_param)
+    Course::Forum::PostNotifier.post_voted(current_user, @post)
```

Comment:
It is actually possible in this case for the the `current_user` to switch from a 'up-vote' to a 'no-vote'. I don't think we should report those as activities cause it isn't really an 'activity'.

Another question is: do we want to differentiate between 'upvotes' and 'downvotes'? Or just state 'votes' will do? 

It might be possible to move this into the `Course::Discussion::Post::Vote` model instead; where we can do a notification callback for `after_create` and `after_update`, since those take care of upvotes and downvotes, while transiting from a up/downvote to a no-vote would just destroy that particular record (and will not call a notifier).